lib/easypost/services/billing.rb

```diff
@@ -62,11 +62,13 @@ def get_payment_method_info(priority)
     end
 
     payment_method_id = payment_methods[payment_method_to_use]['id']
+    payment_method_object_type = payment_methods[payment_method_to_use]['object']
+
+    unless payment_method_object_type.nil?
```

Comment:
suggestion: I don't think we need this nil check here. We needed it before because `start_with` couldn't be called on a nil, but here we are just accessing a variable and checking it against another string. This could reduce all the indentations by one level